src/index.ts

```diff
@@ -241,14 +241,11 @@ export default class PrettyFormatter extends SummaryFormatter {
     }
   }
 
-  private onTestCaseFinished(_testCaseFinished: messages.ITestCaseFinished) {
+  private onTestCaseFinished(_testCaseFinished: messages.TestCaseFinished) {
     this.newline()
   }
 
-  private renderTags(
-    indent: number,
-    tags: messages.GherkinDocument.Feature.ITag[]
-  ) {
+  private renderTags(indent: number, tags: readonly { name: string }[]) {
```

Comment:
Just had to change this to the bare minimum interface shared by `Tag` and `PickleTag`